Dockerfile

```diff
@@ -22,11 +22,11 @@ RUN --mount=type=bind,source=.,target=.,readonly\
 RUN --mount=type=bind,source=.,target=.,readonly\
     --mount=type=cache,target=/root/.cache/go-build \
     --mount=type=cache,target=/go/pkg/mod \
-    just output_dir=/build build faucet-v2
+    just output_dir=/build build faucet
```

Comment:
Why is this changed?

Reply:
Fixed. i think it's by mistake when i was resolving conflict.